### ISSUE TITLE
November 12, 2023

### ISSUE THREAD

**Issue by kyurikotpq** [kyurikotpq/content-garden#6]
# Daily Tasks
- [x] Morning Pages (10mins + 500 words)
- [ ] 1x LinkedIn Post
- [ ] 1x Instagram Post
- [x] 10x Before-bed headlines

**Comment by kyurikotpq** [comment]
I went to the Medicine Buddha service today. I got curious about the 8th aspiration:

> 第八大愿：愿我来世得菩提时，若有女人，为女百恶之所逼恼，极生厌离，愿舍女身。闻我名已，一切皆得转女成男，具丈夫相，乃至证得无上菩提。([Source](https://baike.baidu.com/item/%E8%8D%AF%E5%B8%88%E4%BD%9B%E5%8D%81%E4%BA%8C%E5%A4%A7%E6%84%BF/1622229))

And (with my poor Mandarin) I originally misinterpreted the aspiration: _If there are females, may they become male_.

Now, after my own research, I realized that I missed the entire front part of the aspiration: 
- If there are females who are vexed by the obstacles of being female, and
- Are disgusted by it, and
- Want to leave their female body
- May they change their gender to male (by hearing Medicine Buddha's name) and may that choice lead directly to enlightenment.

The actual aspiration, translated:

> 8. May those who wish to change gender (from female to male) have that wish be fulfilled. May that choice lead directly to enlightenment. If this does not come to pass, may I not reach enlightenment. ([Source](https://www.padmasambhava.org/sermon/12-aspirations-of-medicine-buddha/))

---

There's so much to unpack here. For one, what are "the obstacles of being female"? There's 3 aspects:
1. Tendency to be jealous, greedy, and other not-wholesome actions in Buddhism
2. Physical pain (menstruation and childbirth)
3. Living in a patriarchal society
 
If you think about it, No. 2 and 3 encourage the not-wholesome actions from No. 1, especially if one is ignorant or not careful about their actions and speech.

As such, it's not about the gender differences in physical appearance, but rather the emotional characteristics/tendencies. Upon reflection, these tendencies seem to be true.
- I've seen it play out in a lot of people in my life.
- Some of the dramas in my life were due to these tendencies (and I've been guilty of starting some dramas myself)
- Is this why most men on dating apps come across as very simple/one-dimensional and underwhelming?

---

There were several other mind-blowing moments, but the 8th aspiration and "人身难得"—"having a human body is rare"—really made me go, "Wait, wait, wait."

Having a human body is rare! Your mother lost 99% of the eggs she was born with—even before she started menstruating. There's a lot of sperm cells in ejaculation but only one can successfully enter the egg cell. Literally so many things can go terribly wrong during 9 months of pregnancy.

If we quote Mel Robbin, the chances of us being born is `one in 400 trillion`.

You could also have been born an animal, where the chances of succumbing to your animalistic (and unwholesome) tendencies are much higher, and your chances of hearing the Dharma are much much lower.

So yeah, these are just my main thoughts from today's session. With awareness comes the power to make changes and improve!

**Comment by kyurikotpq** [comment]
Speaking of the session, I also had another thought:

> Will my prayers truly be enough to help the sick recover fully and the dying have a peaceful death?

(I think something similar was brought up in the sutra as well)

At first, this thought made me afraid—because prayers made with a doubtful heart are essentially ineffective. But then I slowed down and thought about it:
- Buddhism revolves around karma: Wholesome actions reap good outcomes, unwholesome actions reap bad outcomes.
- Karma exists like seeds: You don't plant them and they'll sprout 10cm in 12 hours.

And precisely because of the seed analogy, my prayers might not be enough to help the sick recover fully and the dying have a peaceful death.

> Their suffering exists as a result of their bad karma from _way, way_ back, and they have to pay for it somehow in this lifetime.

If they're near the end of their repayment, then yes, perhaps my prayers can work miracles (I put "perhaps" here because I'm assuming their net negative karma keeps decreasing, which can be a very difficult thing to do for the ignorant).

But also precisely because of the seed analogy, the merits of my prayers might help them in the _future_. The future might not be in this lifetime, but at death (e.g. for better rebirth in the next lifetime).

As humans, we want instant gratification, so we keep forgetting this principle and despair when "our prayers don't work".

We stop praying, we stop practicing, and then the net negative karma increases xD

> And so, I'm not going to be doubtful of the Buddhas and the Dharma going forward.

I think a "do-what-you-can" approach will remind me of the principles I've listed here, and keep despair at bay.

Everyone has their own karmic debt to repay.

Whatever help my prayers can give to them, I hope it will help them.

And just because the help doesn't manifest in this life, doesn't mean it's useless.
